atomic_reactor/inner.py

```diff
@@ -250,18 +259,23 @@ def __init__(self, source, image, prebuild_plugins=None, prepublish_plugins=None
         :param plugin_files: list of str, load plugins also from these files
         :param openshift_build_selflink: str, link to openshift build (if we're actually running
             on openshift) without the actual hostname/IP address
+        :param client_version: str, osbs-client version used to render build json
+        :param buildstep_plugins: dict, arguments for build-step plugins
         """
         self.source = get_source_instance_for(source, tmpdir=tempfile.mkdtemp())
         self.image = image
 
         self.prebuild_plugins_conf = prebuild_plugins
+        self.buildstep_plugins_conf = buildstep_plugins
         self.prepublish_plugins_conf = prepublish_plugins
         self.postbuild_plugins_conf = postbuild_plugins
         self.exit_plugins_conf = exit_plugins
         self.prebuild_results = {}
+        self.buildstep_result = {}
         self.postbuild_results = {}
         self.prepub_results = {}
         self.exit_results = {}
+        self.build_result = None
```

Comment:
Wonder if we really need to add this. Alternatively, if we add this, do we still need self.build_failed, self.build_logs, etc?